src/platform/plugins/shared/controls/public/controls/esql_control/esql_control_selections.ts

```diff
@@ -80,10 +92,22 @@ export function initializeESQLControlSelections(
     )
     .subscribe((result) => {
       if (getESQLSingleColumnValues.isSuccess(result)) {
-        availableOptions$.next(result.values);
+        availableOptions$.next(result.values.map((value) => value));
       }
     });
 
+  // Filter the displayed available options by the current search string
```

Comment:
If we're doing local searching for now, I wonder if we can use `contains` style searching instead of `starts with`?